setup.py

```diff
@@ -16,7 +16,7 @@
       author='liluo',
       author_email='i@liluo.org',
 
-      packages=find_packages(),
+      packages=find_packages(exclude=['tests', 'tests.*', 'samples', 'samples.*']),
```

Comment:
`samples.json`/`samples.py` is required.

Reply:
`'samples', 'samples.*'` here refers to the top level /samples/ directory only, and is needed only to exclude the files in `samples/Python/` because they confuse setuptools into believing there is a top level `samples` *package* to be installed.

Reply:
ping @liluo .  This doesnt effect `linguist/libs/samples.*`

Reply:
ping @liluo